protocol-units/bridge/integration-tests/README.md

```diff
@@ -0,0 +1,128 @@
+```
+
+3. In `config.yaml` replace the `account` of `local_root` with `000000000000000000000000000000000000000000000000000000000a550c18` and skip the faucet step by pressing Enter when prompted. Your `config.yaml` will look like this:
+
+```
+---
+profiles:
+  default:
+    network: Custom
+    private_key: "<some-private-key>"
+    public_key: "0x2b8c073bf4c091649d8fb5c275cacc6c8cf8cb6baaf0d7dffc47216011b6a27d"
+    account: e813b12fc00bed33b54b5652c3bb1cbf12a33080aba9cd12d919b6d65cec6115
+    rest_url: "http://localhost:30731/v1"
+    faucet_url: "http://localhost:30732/"
+  local_root:
+    network: Custom
+    private_key: "0x0000000000000000000000000000000000000000000000000000000000000001"
+    public_key: "0x4cb5abf6ad79fbf5abbccafcc269d85cd2651ed4b885b5869f241aedf0a5ba29"
+    account: 000000000000000000000000000000000000000000000000000000000a550c18
+    rest_url: "http://localhost:30731/v1"
+```
+
+4. List the contents of the `0xa550c18` account:
+
+```
+ movement account list --account 0xA550C18
+```
+
+You should see a large number of `AptosCoin` held by the account:
+
+```
+{
+      "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>": {
+        "coin": {
+          "value": "18446743973709450015"
+        },
+        "deposit_events": {
+          "counter": "1",
+          "guid": {
+            "id": {
+              "addr": "0xa550c18",
+              "creation_num": "2"
+            }
+          }
+        },
+        "frozen": false,
+        "withdraw_events": {
+          "counter": "1",
+          "guid": {
+            "id": {
+              "addr": "0xa550c18",
+              "creation_num": "3"
+            }
+          }
+        }
+      }
+    }
+```
+
+This large number of gas coins signifies that this is the core resource account.
+
+5. Verify that you control the core resources signer by transferring gas tokens to your default account:
+
+```
+movement account transfer --profile local_root --amount 100 --account e813b12fc00bed33b54b5652c3bb1cbf12a33080aba9cd12d919b6d65cec6115
+```
+
+You'll be prompted to submit a transaction. After doing so, if the transaction is successful, you'll get a success message and transaction result.
+
+## Steps to run client integration tests
+
+Now that the core resource signer is configured, you can run the client integration tests.
+
+There are two scripts in `protocol-units/bridge/move-modules`, one for enabling the Atomic Bridge feature flag and one for updating bridge operator.
+
+1. Compile the scripts:
+
+```
+movement move compile --package-dir protocol-units/bridge/move-modules
+```
+
+2. Run the `enable_bridge_feature` script:
+
+```
+ movement move run-script --compiled-script-path protocol-units/bridge/move-modules/build/bridge-modules/bytecode_scripts/enable_bridge_feature.mv --profile local_root
+ ```
```

Comment:
Does this script need to be ran manually? I think the integration test handles this? Maybe it's worth adding a note that these steps are for manual testing and they take place automatically in the integration test

Reply:
Thanks, that was before I had the scripts implemented. Updated the README to a simpler version.